aws-net-workshop/aws-net-workshop/examples_bonus/_04_Lifecycle.cs

```diff
@@ -25,7 +49,11 @@ public static void Main(string[] args)
             }
```

Comment:
This must have carried over from previous commit, but do you want to output "fail" to the console if you don't get 200 back from server?